The Changelog at the end of the tunnel (#48185)
CleverRaven/Cataclysm-DDA: data/changelog.txt

Adds Hot Springs location.
Adds a terrain flag 'NO_SHOOT' to prevent projectiles from striking or going through that tile type.
Enables using a rope to carry large items.

> Adds large dermatik nest, tweak dermatiks.

Reviewer: ```suggestion
Adds large dermatik nest and tweaks dermatiks.
```

Aftershock: Megacity-oriented overmap generation adjustments
Aftershock: Adds new Melee oriented bionics.
[DinoMod] Adds Acrocanthosaurus and overhauls dino baby code.

> [Magiclysm] Adds New Black Dragon Mutation Line.

Reviewer: ```suggestion
[Magiclysm] Adds a new mutation line: Black Dragon.
```

Adjusts fog frequency in black dragon lair plus other (minor) map adjustments.
Adds monsters 'Scutellosaurus', 'Maiasaura', and 'Coelophysis'.
[Magiclysm] Attunements - Special abilities gained by mastering two schools of magic.
Adds monsters 'Pachyrhinosaurus', 'Pentaceratops', and 'Torosaurus'.
[DinoMod] Organizes dinos into three main factions with defined inter-faction behavior.

> [DinoMod] Rebalance Jurassic spawns and monsters for realism, fix predator in-fighting.

Reviewer: ```suggestion
[DinoMod] Rebalances Jurassic spawns and monsters for realism and fixes predator in-fighting.
```

Monsters with larger eyes see farther than smaller variants.
Ankle storage items don't affect feet-related mutations anymore.
Gives the default Survivor profession more weather-appropriate clothes for the default start.
Eating raw meat does not hurt your morale as much if you have traits like Hunter or Predator.
Very-fast-moving zombie animals only attack once per turn.

> Spawn anesthesia in autodocs.
> Rebalance overmap special spawn rates.
> Mycus Fireproofing now also gives heat immunity.
> Tweak Infection impact by lowering fatigue drain and removing pain.
> Adjust ordinary clothing to not meaningfully contribute to protection values.

Reviewer: ```suggestion
Spawns anesthesia supplies in autodocs.
Rebalances spawn rates for overmap specials.
Mycus Fireproofing now also gives heat immunity.
Tweaks Infection impact by lowering fatigue drain and removing pain.
Adjusts ordinary clothing to not meaningfully contribute to protection values.
```

Bashes that do zero damage now can't stun.
Fixes a crash due to dereferencing a null carrier pointer in item::process_tool.
Prevents a seg fault in auto-consume.
Vehicles: Fixes towed vehicle drag by unsetting their parking brakes.
Fixes sarcophagus elevator reactivation.

> Fixes doors that NPC's could not pass without intervention.
> Prevents work bench disassembly from vanishing.
> Fix error message on towed vehicle crashed with towing vehicle

Reviewer: ```suggestion
Fixes doors that NPCs could not pass without intervention.
Prevents disassemblies on a workbench from vanishing.
Fixes errors from crashes between a towed vehicle and its towing vehicle.
```

Bases returned materials from salvaging on mass instead of volume.

> Fixes unable to run message, allow mount to run when your legs are broken.
> Fixes abandon faction camp error.
> Makes NPC fight back while guarding or patrolling.

Reviewer: ```suggestion
Fixes unable to run message when mounted - allows a mount to run when your legs are broken.
Fixes errors from abandoning a faction camp.
Makes NPCs able to fight back while guarding or patrolling.
```

Fixes use-after-free error in sounds::process_sound_markers(player).

> Makes Zombie technician pull metal weapon attack drop items at technician's feet.
> Fix bad memory access in drop_activity_actor::do_turn().

Reviewer: ```suggestion
Makes the Zombie technician's pull metal weapon attack drop items at the technician's feet.
Fixes errors due to a bad memory access in drop_activity_actor::do_turn().
```

Prevents slow processing of NPC overmap travel when `goal` is unreachable.
Fixes the performance bug caused by `verify_vehicle_cache` recalculation.
Adds lightweight crafting inventory cache to improve NPC construction processing time.

> Reduces save size by skipping serialization of some item pockets and relic_data.
> Adds RLE compression for json ser/deser of items colony. This reduces savefile size.
> Improves performance of initializing achievements_tracker.

Reviewer: ```suggestion
Reduces save size by skipping serialization of some item pockets and relic data.
Adds RLE compression for JSON serialization/deserializations of data on items, reducing savefile size.
Improves performance of initializing the achievements tracker.
```

Adds typesafe angle type.
Adds check that Generic Guns migrates all guns, magazines, and ammo.
Enables JSON member visitation checks outside of tests (better consistency checking).

> Display multi-line json error messages with github action.
> Python tool to easily translate map coordinates.
> Enable json check for mods in github matrix.

Reviewer: ```suggestion
Displays multi-line JSON error messages correctly when building on Github.
Python tool added to easily translate map coordinates.
Enables a JSON check for mods in the Github build matrix.
```

> Have and check correct season length for tests.

Reviewer: ```suggestion
Uses (and checks for using) the correct season length for tests.
```

Adds problem matchers for json errors, compiler errors, DebugLog emissions, and Catch2 errors.
Adds check enforcing almost-never-auto convention.
Adds a GitHub workflow running flake8 to check the Python scripts.

> Unify code style in CMakeLists.txt files.

Reviewer: ```suggestion
Consistently styles code in CMakeLists.txt files.
```

Adds the start of a translator glossary and explains the etymology of Exodii.
Uses class translation in many more places for more flexible translation support.
Infrastructure: Automates translation template push.

> Allows selection of Arabic, Czech, Danish, Greek, Indonesian, Icelandic, Italian, Norwegian, Dutch, Serbia, Turkish, Ukrainian languages in game.

Reviewer: ```suggestion
Enables using additional languages (Arabic, Czech, Danish, Dutch, Greek, Indonesian, Icelandic, Italian, Norwegian, Serbian, Turkish, and Ukrainian) with the game.
```
